src/Algebra/Solver/Monoid/Tactic.agda

```diff
@@ -0,0 +1,52 @@
+------------------------------------------------------------------------
+-- The Agda standard library
+--
+-- A solver for equations over monoids
+------------------------------------------------------------------------
+
+{-# OPTIONS --cubical-compatible --safe #-}
+
+open import Algebra.Bundles using (Monoid)
+import Algebra.Solver.Monoid.Expression as Expression
+
+module Algebra.Solver.Monoid.Tactic {a c ℓ} (M : Monoid c ℓ)
+  (open Monoid M using (rawMonoid; setoid; _≈_))
+  (open Expression rawMonoid using (Expr; Env; var; ⟦_⟧; NormalAPI))
+  (N : NormalAPI {a}) where
+
+open import Data.Maybe.Base as Maybe
+  using (Maybe; From-just; from-just)
+open import Data.Nat.Base using (ℕ)
+open import Function.Base using (_∘_; _$_)
+open import Relation.Binary.Consequences using (dec⇒weaklyDec)
+open import Relation.Binary.PropositionalEquality.Core using (_≡_; cong)
+import Relation.Binary.Reflection as Reflection
+
+open NormalAPI N
+open module R = Reflection setoid var ⟦_⟧ (⟦_⟧⇓ ∘ normalise) correct public
+  using (solve; _⊜_)
+
+private
+  variable
+    n : ℕ
+
+
+------------------------------------------------------------------------
+-- Tactic
+
+-- We can also give a sound, but not necessarily complete, procedure
```

Comment:
What does the `also` refer to here?

Reply:
Comment copied over from original. Have rejigged the module contents to make the exports more explicit, and hopefully the "also" now makes more sense?